src/Factories/PDOFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace App\Factories;
+
+
+use PDO;
+use Psr\Container\ContainerInterface;
+
+class PDOFactory
```

Comment:
Love how much more consistent this is gonna be for teaching - always seemed a little weird in the past that we just randomly jammed db code in the DIC